src/index.ts

```diff
@@ -51,6 +52,17 @@ export class TestxServer {
     };
 
     const apolloServer = new ApolloServer({ typeDefs, resolvers, context });
-    return apolloServer;
+
+    // Sets PORT as a random port available in the range 29170 - 29998
+    const ports = await portastic.find({ min: 29170, max: 29998 });
+    const PORT = ports[Math.floor(Math.random() * 100)];
```

Comment:
Upper case names for variables should be used only for global constants
```suggestion
    const port = ports[Math.floor(Math.random() * 100)];
```